lib/data_table.rb

```diff
@@ -467,7 +467,7 @@ def get_resultset_and_count(params)
 
         query = self.query!(params)
         query.offset = params.fetch('skip', 0).to_i
-        query.limit = params.fetch('take', @options[:paging][:pageSize]).to_i
+        query.limit = params.fetch('take', params[:take]).to_i if params[:take]
```

Comment:
The reason for calling `params.fetch('take'` seems a bit pointless with the `if params[:take]` check.

Reply:
it's just so the query.limit is not assigned.